ctdcal/process_ctd.py

```diff
@@ -1937,23 +1938,16 @@ def export_ct1(df, ssscc_list):
                     print(col + " missing, filling with -999s")
                     df[col] = -999
 
-    # load other necessary params
-    # TODO: this will change when config.py improves, find different method
-    expocode = cfg.cruise["expocode"]
-    section_id = cfg.cruise["sectionid"]
-    ctd = cfg.ctd_serial
-    out_dir = cfg.directory["pressure"]
-    p_col = 'CTDPRS'
-    stacst_col = 'SSSCC'
-    logFile = cfg.directory["logs"] + 'cast_details.csv'
-
-    df[stacst_col] = df[stacst_col].astype(str).copy()
-
+    df["SSSCC"] = df["SSSCC"].astype(str).copy()
```

Comment:
Not todays problem nor a code review... but be aware that the WOCE manual only requires the "cast" to be an integer, the "STNNBR" is explicitly allowed to have not number stuff in it.

This is why the bottle database used strings for these... but it's also why we went crazy on that geotraces cruises with their "half" and "thirds" stations (eg. station 12.5 or 12.3 and 12.6)

Reply:
good to know, thanks!
I _think_ the code can be setup such that it's agnostic about "SSSCC" format by leaving it as _str_ until the very end, and then have human intervention to fix up the file output if needed

Reply:
This will be a hard one... I think a lot of the analysis computers assume numbers for everything (salt, oxygen, nuts, pretty sure the total inorganic carbon). So on a cruise where this is happening, you'll always need to have some mapping between what science wants to call their stations and what the old computer can output.

What you do want to avoid if you can, is code that tries to get the "station" or "cast" by doing math on a "ssscc" string e.g. `cast = ssscc % 100` or `station = floor(ssscc/100)`. I've seen the NOAA DIC folks get bit by this, it's even a wrong assumption in Liqing's fancy data "manuscript".

Reply:
This is also why making a break from "SSSCC" so that there's a `station` field and a `cast` field can make things more flexible in the code.